pkg/lua/include/lua_builtin.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+/**
+ * @ingroup  pkg_lua
+ * @ingroup  pkg
+ * @ingroup  sys
+ * @{
+ * @file
+ *
+ * @brief   Definitions for including built-in modules.
+ * @author  Juan Carrano <j.carrano@fu-berlin.de>
+ *
+ */
+
+#ifndef LUA_BUILTIN_H
+#define LUA_BUILTIN_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
```

Comment:
double empty line here

Reply:
We really need a tool to find these.

Reply:
that's the idea of #8519. Hope it gets into master soon

Reply:
This is corrected